webapp/src/components/viewHeader/viewHeader.tsx

```diff
@@ -59,32 +75,71 @@ const ViewHeader = React.memo((props: Props) => {
 
     const hasFilter = activeView.fields.filter && activeView.fields.filter.filters?.length > 0
 
+    const isOnboardingBoard = props.board.title === OnboardingBoardTitle
```

Comment:
All this code, is again, like 50 lines of code to add to one component to show decide to show or not the tooltip, I don't think it belongs to the main responsaibility of this component, it is making the component hard to understand.

Reply:
The logic is different in different places. We still need to get the tour state in the current component because it's stored in the redux store, and a utility function can't access it.

Reply:
Let me try one thing. It might work.

Reply:
Done.